Wikipedia/Code/WMFFeedContentSource.m

```diff
@@ -103,6 +103,23 @@ - (void)fetchContentForDate:(NSDate *)date force:(BOOL)force completion:(void (^
         }
         success:^(WMFFeedDayResponse *_Nonnull feedDay) {
 
+            NSString *key = [WMFFeedDayResponse WMFFeedDayResponseMaxAgeKey];
+            NSFetchRequest *request = [WMFKeyValue fetchRequest];
+            request.predicate = [NSPredicate predicateWithFormat:@"key == %@", key];
+            request.fetchLimit = 1;
+            NSManagedObjectContext *moc = self.userDataStore.viewContext;
```

Comment:
It's possible this isn't the main thread, so `viewContext` shouldn't be used - you can move all of this into  `- (void)saveContentForFeedDay:(WMFFeedDayResponse *)feedDay pageViews:(NSDictionary<NSURL *, NSDictionary<NSDate *, NSNumber *> *> *)pageViews onDate:(NSDate *)date inManagedObjectContext:(NSManagedObjectContext *)moc completion:(dispatch_block_t)completion` and use the provided context

Reply:
(inside of the `performBlock:^{`)